ejb3/src/main/resources/schema/jboss-ejb-client_1_4.xsd

```diff
@@ -162,6 +162,37 @@
         </xsd:attribute>
     </xsd:complexType>
 
+    <xsd:complexType name="http-connectionsType">
+        <xsd:choice>
```

Comment:
Need min and max bound for choice:
        <xsd:choice minOccurs="0" maxOccurs="unbounded">